### PR TITLE
updates docs

### PR DESCRIPTION
**Pull request by jonborchardt**
minimal docs update to explain new lib

### CODE REVIEW

File: components/link/index.en-US.md [hunk 1]
```diff
@@ -9,16 +9,28 @@ cover: https://allenai.org/favicon.ico
 Varnish has default styles for `a[href]` links and provides a base style `contrastLinkColorStyles`
 for use on dark backgrounds.
 
-## Recommended InternalLink
+## react-router
+
+`Varnish` does not depend on `react-router`.  If you wish to use `react-router` in your app, you can include `@allenai/varnish-react-router` for access to styled router links and other `react-router` dependent components.
```
Comment (codeviking) [inline components/link/index.en-US.md:14]: Can we link to `@allenai/varnish-react-router`'s README here so they know how to install / add it?

File: components/react-router/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+category: Components
+type: Extensions
+title: varnish-react-router
+cols: 2
+cover: https://allenai.org/favicon.ico
+---
+
+`Varnish` does not depend on `react-router`.  If you wish to use `react-router` in your app, you can include `@allenai/varnish-react-router` for access to styled router links and other `react-router` dependent components.
```
Comment (codeviking) [inline components/react-router/index.en-US.md:9]: Link here too?

File: components/link/index.en-US.md (hunk 1 above)
Comment (jonborchardt) [reply components/link/index.en-US.md:14]: done

File: components/react-router/index.en-US.md (hunk 2 above)
Comment (jonborchardt) [reply components/react-router/index.en-US.md:9]: done

**Pull request merged**